docs/jade-getting-started.md

```diff
@@ -157,21 +157,21 @@ and configure Docker to connect to the appropriate Google Cloud endpoint when
 necessary:
 
 ```
-brew cask install google-cloud-sdk
+brew install --cask google-cloud-sdk
 gcloud auth login
 gcloud auth application-default login
 gcloud auth configure-docker
 ```
 
 10. [IntelliJ IDEA](https://www.jetbrains.com/idea/) is an integrated development
 environment (IDE) for Java. There are two versions available: **Ultimate** (paid)
-and **Community** (open-source). The **Community** edition has all the features
-needed for development:
+and **Community** (open-source). We recommend the Ultimate Edition for its database navigation capabilities:
 
 ```
-brew cask install intellij-idea-ce
-open -a "IntelliJ IDEA CE"
+brew install --cask intellij-idea
+open -a "IntelliJ IDEA"
 ```
+Connect to the Broad VPN and use http://jetbrains.broadinstitute.org:8080 as the license server.
```

Comment:
Nice catch! I do wonder if we want to publish this license server address on our public repo, though. I assume it's not a big deal since you can't use it until you're on the broad VPN, but not sure.

Reply:
Yeah, I don't think we want to include that since some folks don't use the ultimate edition either. Can you rebase to remove this?

Reply:
@fboulnois  - motivation behind this change - Definitely still optional, but Muscles and I have recommended using the ultimate edition in order to use intellij's database functionality.

Reply:
I think there's no problem publishing this hostname publicly. Most of the time I use ultimate, but I have CE installed too in case I want to work without getting on the VPN. This wasn't much of an issue when we went into the office every day since office Wifi is inside the VPN subnet.